toolchain/parse/dump_id.cpp

```diff
@@ -26,4 +27,19 @@ auto DumpIdImpl(const Tree& tree, NodeId node_id) -> void {
   llvm::errs() << ")";
 }
 
+// A set of DumpId() overloads that dump an object to stderr, useful for
```

Comment:
In Lex, this is moved to the header, should this comment move to the header here too? Or can we make it consistent either way

Reply:
Added a more complete comment to each header.

Reply:
Also a mention in docs/